### PR TITLE
adding seed

### PR DESCRIPTION
**Pull request by knudsem**
Adding faker Gem, Adding db:seed with faker

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -17,3 +17,28 @@
...
+        user_id: user.id,
+        name: Faker::Name.first_name,
+        sqm: rand(0..200),
+        address: Faker::Address.secondary_address,
+        price: rand(100.2...8000.1),
+        photo: "https://picsum.photos/200/300"
+      )
+  end
+end
+puts 'Finished!'
```
Comment (ronitalvi) [inline db/seeds.rb:50]: you need to save storage

File: db/seeds.rb [hunk 2]
```diff
@@ -17,3 +17,28 @@
...
+    password: Faker::Code
+  )
+  user.save!
+  10.times do
+    storage = Storage.new(
+        user_id: user.id,
+        name: Faker::Name.first_name,
+        sqm: rand(0..200),
+        address: Faker::Address.secondary_address,
+        price: rand(100.2...8000.1),
```
Comment (ronitalvi) [inline db/seeds.rb:44]: storage needs to have description

File: db/seeds.rb [hunk 3]
```diff
@@ -17,3 +17,28 @@
...
+
+
+
+puts 'Creating 10 fake users...'
+10.times do
+  user = User.new(
+    email:    Faker::Internet.email,
+    name:   Faker::Movies::StarWars.character,
+    country_code:  rand(1..50),
+    password: Faker::Code
```
Comment (ronitalvi) [inline db/seeds.rb]: 6times 0 or 123456

**Comment by knudsem** [comment]
good work bro

**Pull request merged**